### ISSUE TITLE
Failure to transform cuboid

### ISSUE THREAD

**Issue by ngffy** [jan-mue/geometer#23]
```python
from geometer import *

cuboid = Cuboid(Point(0,0,0),Point(1,0,0),Point(0,1,0),Point(0,0,1))
translate = translation(1,0,0)

translate.apply(cuboid)
```

On my machine, this code results in an error:

> Traceback (most recent call last):
> 
>   File "bug.py", line 6, in <module>
>     translate.apply(cuboid)
>   File "/usr/lib/python3.8/site-packages/geometer/transformation.py", line 133, in apply
>     return type(other)(other.array.dot(self.array.T))
> TypeError: \_\_init\_\_() missing 3 required positional arguments: 'b', 'c', and 'd'

It looks to me like the problem is since `type(other)` is a cuboid in this instance, the cuboid constructor is called, but arguments for the polytope constructor are given.

It seems like a workaround is to cast the cuboid to a polyhedron before trying to transform it.

**Comment by jan-mue** [comment]
I can confirm the bug and already implemented an easy fix for this. Now applying a transformation to a Cuboid will simply return a Polytope. This clearly isn't an ideal solution for this issue and I might have to think of a better way to apply the transformations.
Thanks for reporting this bug!

**Comment by jan-mue** [comment]
I just released version 0.2.1 that should solve this issue.
